examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx

```diff
@@ -1,9 +1,9 @@
+import { Field, PublicKey } from 'o1js';
 import { useEffect, useState } from 'react';
-import './reactCOIServiceWorker';
-import ZkappWorkerClient from './zkappWorkerClient';
-import { PublicKey, Field } from 'o1js';
 import GradientBG from '../components/GradientBG.js';
 import styles from '../styles/Home.module.css';
+import './reactCOIServiceWorker';
+import ZkappWorkerClient from './zkappWorkerClient';
 
 let transactionFee = 0.1;
 const ZKAPP_ADDRESS = 'B62qknCphH8ikE9Bu46dFgamUeYRcE9FtKr5ahh5tcJDf7ibRXJdzWV';
```

Comment:
This can be for another PR, but we should deploy this contract to `devnet`

Reply:
I'll deploy one, thanks.

Reply:
`Add` contract deployed on `Devnet` at: `B62qpXPvmKDf4SaFJynPsT6DyvuxMS9H1pT4TGonDT26m599m7dS9gP`.